File: lec002/lec02.java
import java.util.*;

class lec02{
    // getting started again.
    public static void main(String[] args){
        
        Scanner scn=new Scanner(System.in);

        System.out.println("enter 1st number");
        int a=scn.nextInt();
        System.out.println("enter 2nd number");
        int b=scn.nextInt();

        int res=a+b;
        System.out.println("your res is="+res);


        // if else conditions

        int marks=scn.nextInt();

        if(marks>90){
            System.out.println("I will take science");
        }else{
            System.out.println("I will not study");
        }

        // loop (print till given value)

        int value=scn.nextInt();

        for(int i=0;i<=value;i++)
        {
            System.out.print(i+" ");
        }



    }
}

File: lec002/lec02.cpp
#include<iostream>
using namespace std;

// is a number prime?

int main(){

    int n=0; cout<<"enter your number"<<endl;
    cin>>n;
    bool isprime=true;  // assumption.
    for(int i=2;i*i<=n;i++){
        if(n%i==0){
            isprime=false;   break;
        }
    }
    if(isprime){
        cout<<"prime";
    }else{
        cout<<"not prime";
    }



        

    return 0;
}